### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by Pughal77**
Update quick guide in User Guide

Updated `Ui.png` to inlcude our current Ui
Added link to our github release

### CODE REVIEW

File: src/test/data/ExportCommandTest/addressbookdata/addressbooktest.csv
Comment (danielleloh) [inline src/test/data/ExportCommandTest/addressbookdata/addressbooktest.csv:1]: sorry, these files should have been .gitignored edit :") perhaps we can remove these files?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -15,13 +15,17 @@ Avengers Assemble (AA) is a **desktop app for managing contacts, optimized for u
...
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your computer.
+1. Ensure you have `Java 11` or above installed in your computer.
 
-1. Download the latest `avengersassemble.jar`.
+1. Download the latest `avengersassemble.jar` [here](https://github.com/AY2324S2-CS2103T-T10-1/tp/releases/tag/v1.2).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AA.
+1. Copy the file to the folder you want to use as the _home folder_ for our application.
```
Comment (jayllo-c) [inline docs/UserGuide.md:22]: Changes look more straightforward for users now

**Review by jayllo-c — COMMENTED** [review]
Good job on the changes, just need to ensure that the screenshots don't conflict

**Review by delishad21 — COMMENTED** [review]
Ui image changes currently conflicts with lengkhais PR. Shoud remove it to remove merge conflicts.

**Pull request merged**